app_dart/lib/src/model/firestore/commit_tasks_status.dart

```diff
@@ -73,9 +73,19 @@ class FullTask {
   final Task task;
   final List<int> buildList;
 
-  Map<String, dynamic> toJson() {
-    return <String, dynamic>{
-      'Task': task.facade,
+  Map<String, Object?> toJson() {
+    return {
+      'Task': {
+        'DocumentName': task.name,
```

Comment:
Should these be `Task.fieldBringup` or was the UpperCase on purpose?

Reply:
Chat: because this is the frontend data schema, and we'll eventually delete this and use just one schema. You know, like sane people.